### ISSUE TITLE
Ionic native documentation missing points.

### ISSUE THREAD

**Issue by harshzalavadiya** [ionic-team/ionic-docs#328]
In new Ionic Native docs instructions are missing about what environment variables that are needed when installing plugin for example

**Environment Variables**
[image-picker](https://ionicframework.com/docs/native/image-picker/) requires `PHOTO_LIBRARY_USAGE_DESCRIPTION` environment variable can be seen [here](https://github.com/Telerik-Verified-Plugins/ImagePicker) when otherwise it crashes instantly when triggering plugin in android

**improper import instructions**
- `import { plugin } from @ionic/native/plugin/ngx` missing `ngx`
- I know instructions are there on `docs/native` but there's no link that says that plus if some plugins use separate import pattern like `ionic-native-google-maps` which is missing is using non `ngx` import
- plus like beta docs how to import plugin in `app.module.ts` link is also missing. that may confuse new users and is also handy for less experienced users

it will be really nice to have above changes.

and thanks for making such an great documentation.

**Comment by camwiegert** [comment]
@harshzalavadiya Thanks for these suggestions! I'll need to look into whether we can get environment variable info from the plugin source or if this is something we'll need to address for each plugin individually.

**Comment by harshzalavadiya** [comment]
Yes, I understand it's not an small task but older beta documentation had almost everything covered and it will be really nice to have so no developer has to leave beta documentation and read readme of project.

if any help needed i can help collecting data over the weekend

**Comment by jcesarmobile** [comment]
closing since latest plugin docs are for Capacitor only, so won't be changed for previous versions since they are legacy docs now
The official docs are https://danielsogl.gitbook.io/awesome-cordova-plugins
